cluster/s3-nohealth.yaml

```diff
@@ -1,68 +0,0 @@
-#
```

Comment:
This file is used in 
https://github.com/scp756-221/term-project-raccoon-fury/blob/5441bc43792525f5a56bee2280179d4901320816/k8s-tpl.mak#L100

Either we take away s3 from health-off or keep this file

Reply:
The file will be generated from the template when we call `make -f k8s-tpl.mak templates`. It is (removed then) set to be ignored because we have different IDs as the image names.

Reply:
[Here](https://github.com/scp756-221/term-project-raccoon-fury/pull/50/commits/0bd812cfa8adfd9ff3bf3caf27b210f66b720dcf) is the commit to set `.gitignore`

Reply:
oh i see. totally forgot the tpl file